modules/i18n/src/translations/fr-FR/portals/console.ts

```diff
@@ -3647,7 +3647,8 @@ export const console: ConsoleNS = {
                 templates: {
                     enterprise: {
                         validation: {
-                            name: "Merci d'entrer un nom valide"
+                            name: "Merci d'entrer un nom valide",
+                            invalidName: "Merci d'entrer un nom valide"
```

Comment:
Inconsistent with the en-US `invalidName`

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2516